app/models/host.rb

```diff
@@ -8,6 +8,8 @@ class Host < ApplicationRecord
                 only_explicit: true
   scoped_search on: :compliance_score, ext_method: 'filter_by_compliance_score',
                 only_explicit: true
+  scoped_search on: :has_test_results, ext_method: 'has_test_results',
+                only_explicit: true
```

Comment:
We should limit available operators:

```suggestion
                only_explicit: true, operators: ['=', '!=']
```